### PR TITLE
feat: migrate inspector toolbox to blade, hide type additions to subsets

### PR DESCRIPTION
**Pull request by wescopeland**
This PR migrates the achievement inspector toolbox component (`/achievementinspector.php?g=1`) to Blade. It also hides the "Set Selected to Progression" and "Set Selected to Win Condition" buttons if the game is either a subset or a test kit.

In a subsequent PR, I will also adjust the logic for these cases in _achievementinfo.php_ and our API calls.

### CODE REVIEW

File: resources/views/platform/components/developer/inspector-toolbox.blade.php [hunk 1]
```diff
@@ -0,0 +1,167 @@
...
+    function toolboxComponent() {
+        return {
+            areCodeRowsHidden: true,
+
+            /**
+             * @param {'flag' | 'type'} property
+             * @param {3 | 5 | 'progression' | 'win_condition' | null} newValue
+             * @param {number} selectedCount
+             */
+            getConfirmMessage(property, newValue, selectedCount) {
```
Comment (Jamiras) [inline resources/views/platform/components/developer/inspector-toolbox.blade.php:29]: I see this in the page source, but I get the following console error trying to set the type of an achievement or promote/demote one:
```
app-d237b129.js:1 Alpine Expression Error: getConfirmMessage is not defined

Expression: "updateAchievementsProperty('type', 'progression')"

 <button class=​"btn" @click=​"updateAchievementsProperty('type', 'progression')​" fdprocessedid=​"bok2k2">​ Set Selected to Progression ​</button>​flex
Fe @ app-d237b129.js:1
(anonymous) @ app-d237b129.js:5
Promise.catch (async)
(anonymous) @ app-d237b129.js:5
mo @ app-d237b129.js:1
(anonymous) @ app-d237b129.js:5
o @ app-d237b129.js:5
(anonymous) @ app-d237b129.js:5
(anonymous) @ app-d237b129.js:5
app-d237b129.js:5 Uncaught ReferenceError: getConfirmMessage is not defined
    at Proxy.updateAchievementsProperty (achievementinspector.php?g=4381:1264:35)
    at eval (eval at <anonymous> (app-d237b129.js:5:640), <anonymous>:3:32)
    at app-d237b129.js:5:979
    at mo (app-d237b129.js:1:4883)
    at app-d237b129.js:5:35726
    at o (app-d237b129.js:5:25843)
    at app-d237b129.js:5:26832
    at HTMLButtonElement.<anonymous> (app-d237b129.js:5:25865)
```

File: resources/views/platform/components/developer/inspector-toolbox.blade.php [hunk 2]
```diff
@@ -0,0 +1,167 @@
...
+                Set Selected to Win Condition
+            </button>
+        @endif
+
+        <button
+            class="btn"
+            @click="updateAchievementsProperty('type', null)"
+        >
+            Set Selected to No Type
+        </button>
```
Comment (Jamiras) [inline resources/views/platform/components/developer/inspector-toolbox.blade.php:159]: I assume the "Set Selected to No Type" was intentionally not tied to the `$canHaveTypes`? 

Similarly, the "Type: none" is still present in the individual rows.

File: public/achievementinspector.php [hunk 3]
```diff
@@ -306,43 +227,41 @@ function toggleAllCodeRows() {
...
+        $modificationLevel = 'partial';
+    } elseif ($fullModifyOK) {
+        $modificationLevel = 'full';
     }
 
-    echo "<a class='btn w-full flex justify-center py-2' href='/achievementinspector.php'>Back to List</a></p></div><br>";
+    $canHaveTypes = (
+        mb_strpos($gameTitle, "[Subset") === false
+        && mb_strpos($gameTitle, "~Test Kit~") === false
+    );
```
Comment (Jamiras) [inline public/achievementinspector.php:242]: This should also exclude Events

File: resources/views/platform/components/developer/inspector-toolbox.blade.php (hunk 1 above)
Comment (wescopeland) [reply resources/views/platform/components/developer/inspector-toolbox.blade.php:29]: Accidentally broke this during a last-minute refactor. Resolved in latest.

File: resources/views/platform/components/developer/inspector-toolbox.blade.php (hunk 2 above)
Comment (wescopeland) [reply resources/views/platform/components/developer/inspector-toolbox.blade.php]: I did leave it intentionally enabled/visible, but on second thought it doesn't make much sense to do so.

After we ship, we can run this query: https://discord.com/channels/476211979464343552/1174126386349342901 and we shouldn't have the problem anymore. I've hidden this button on latest.

File: public/achievementinspector.php (hunk 3 above)
Comment (wescopeland) [reply public/achievementinspector.php]: Updated in latest to exclude games with console ID 101.

**Review by Jamiras — APPROVED** [review]

**Pull request merged**